zebra-chain/src/block/header.rs

```diff
@@ -118,3 +123,59 @@ pub struct CountedHeader {
     pub header: Header,
     pub transaction_count: usize,
 }
+
+// Includes the 32-byte nonce and 3-byte equihash length field.
```

Comment:
Might as well make this a doc comment.
```suggestion
/// The serialized size of a Zcash block header.
///
/// Includes the equihash input, 32-byte nonce, 3-byte equihash length field, and equihash solution.
```